webapp/src/components/styles/ChatContainer.css

```diff
@@ -82,19 +82,20 @@ ul, li {
 }
 
 p {
-  max-width: 500px;
+  /*max-width: 500px;*/
```

Comment:
ideally we should just delete this if it's no longer required

Reply:
Fixed, thank you!